igdiscover/igdiscover.yaml

```diff
@@ -187,7 +187,8 @@ pre_germline_filter:
   cdr3_shared_ratio: 0.8     # Maximum allowed CDR3_shared_ratio
   unique_d_ratio: 0.3        # Minimum Ds_exact ratio between alleles
   unique_d_threshold: 10     # Check Ds_exact ratio only if highest-expressed allele has at least this Ds_exact count
-
+  # Maximum number of allowed differences from database
```

Comment:
I understand why you want to write the comment above the setting, but it needs to be next to the setting for consistency.

If you want to have the comments above, you should suggest this in a different PR.